squirrel.worker/pom.xml

```diff
@@ -1,135 +1,180 @@
-			
-			<dependency>
-			    <groupId>org.apache.any23</groupId>
-			    <artifactId>apache-any23-encoding</artifactId>
-			    <version>2.2</version>
-			</dependency>
-			
-			<dependency>
-			    <groupId>org.apache.any23</groupId>
-			    <artifactId>apache-any23-mime</artifactId>
-			    <version>2.2</version>
-			</dependency>
-			
-			<dependency>
-			    <groupId>org.apache.tika</groupId>
-			    <artifactId>tika-parsers</artifactId>
-			    <version>1.19.1</version>
-			</dependency>
-
-        <!-- ~~~~~~~~~~~~~~~~~~~ Testing ~~~~~~~~~~~~~~~~~~~~~~ -->
-        <!-- JUnit -->
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.dice-research</groupId>
-            <artifactId>squirrel.mockup</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <!-- ~~~~~~~~~~~~~~~~~~~ End Testing ~~~~~~~~~~~~~~~~~~~~~~ -->
-
-        <!-- ~~~~~~~~~~~~~~~~~~~ Logging ~~~~~~~~~~~~~~~~~~~~~~ -->
-        <!-- slf4j: Logging API -->
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-        </dependency>
-        <!-- log4j binding -->
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>apache-log4j-extras</artifactId>
-        </dependency>
-        <!-- make Logging with java.util.Logging possible -->
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>jul-to-slf4j</artifactId>
-        </dependency>
-        <!-- ~~~~~~~~~~~~~~~~~~~ End Logging ~~~~~~~~~~~~~~~~~~~~~~ -->
-    </dependencies>
-
-    <build>
-        <finalName>${project.artifactId}</finalName>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-            </plugin>
-        </plugins>
-    </build>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.dice-research</groupId>
+		<artifactId>squirrel</artifactId>
+		<version>0.4.0</version>
+	</parent>
+	<artifactId>squirrel.worker</artifactId>
+	<packaging>jar</packaging>
+	<name>Squirrel Worker</name>
+
+
+
+	<dependencies>
+		<dependency>
+			<groupId>org.dice-research</groupId>
+			<artifactId>squirrel.api</artifactId>
+			<!-- <exclusions> <exclusion> <groupId>org.apache.httpcomponents</groupId> 
+				<artifactId>httpclient-cache</artifactId> </exclusion> </exclusions> -->
+		</dependency>
+
+		<dependency>
+			<groupId>eu.trentorise.opendata</groupId>
+			<artifactId>jackan</artifactId>
+			<version>0.4.2</version>
```

Comment:
Versions should be defined in the parent pom as described above.

Reply:
These are dependencies used only on this module, and will be in the future. It is really necessary to do that?

Reply:
Yes. It may look stupid at a first glance. However, I would like to have an easy to follow rule (every dependency should be managed in the parent pom) instead of having a rule but several exceptions for different modules :wink:

Reply:
done